website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -1100,7 +1100,7 @@ tree, [2]
 Thus, `GroupByKey` represents a transform from a multimap (multiple keys to
 individual values) to a uni-map (unique keys to collections of values).
 
-Using <span class="language-java">`GroupByKey`</span> is straightforward:
+<span class="language-java">Using `GroupByKey` is straightforward:
```

Comment:
@denisecase Sorry, just this line, 1103, should be wrapped in `language-java` class.